consul-secure-acl-vault/step3.md

```diff
@@ -10,13 +10,13 @@ Once the ACL system is initialized, if you check Consul logs again you should fi
  [WARN]  agent: Coordinate update blocked by ACLs: accessorID=00000000-0000-0000-0000-000000000002
 ```
 
-These warnings are ACL logs from which you can infer the following: 
-- one node is trying to update its info in your Consul datacenter and gets denied access by the ACL system.
-- the node is not using any token to perform the request (because `00000000-0000-0000-0000-000000000002` is the anonymous policy used when no token is presented)
+These warnings indicate that: 
+- the agent is trying to update its info in your Consul datacenter, but denied by the ACL system because it does not have the correct permissions.
+- the agent does not have an assigned token to perform the request (because `00000000-0000-0000-0000-000000000002` is the anonymous policy used when no token is presented)
 
-First steps towards a more fine grained ACL configuration is to create tokens for the server nodes so that they could interact properly with the rest of the Consul datacenter, without assigning them an administrative token.
+The first step towards a more fine grained ACL approach is to create individual tokens for the server agents, so that they can interact properly with the rest of the Consul datacenter, without assigning them an administrative token.
```

Comment:
```suggestion
The first step towards a more fine grained ACL approach is to create individual tokens for the server agents so that they can interact properly with the rest of the Consul datacenter without being assigned an administrative token.
```